media-gfx/openmesh/openmesh-11.0.0.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+MY_PN="OpenMesh"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Generic data structure to represent and manipulate polygonal meshes"
+HOMEPAGE="https://www.graphics.rwth-aachen.de/software/openmesh/"
+SRC_URI="https://www.graphics.rwth-aachen.de/media/openmesh_static/Releases/$(ver_cut 1-2)/${MY_P}.tar.bz2"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="BSD"
+SLOT="4"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc gui static-libs test"
```

Comment:
I don't really see the purpose of static-libs here - we can probably just drop it?

Reply:
I'm ok with that. Patch modified for shared only. cmake arg is initially only for win32 and was "propagate" for use of static-libs useflag. It's removed now.